project/assets/main/world/restaurant/floor-carpet-sheet.png.import

```diff
@@ -2,15 +2,15 @@
 
 importer="texture"
 type="StreamTexture"
-path="res://.import/floor-brown.png-c9b19e63451bc9c52431b4fbdf9b0d3a.stex"
+path="res://.import/floor-carpet-sheet.png-c9b8a7a20e46aad4e0c2b78a6720ace5.stex"
```

Comment:
Document this change in the commit comment.

Reply:
Updated commit comment, `Renamed 'floor-brown' to a more descriptive 'floor-carpet'`